packages/block-library/src/separator/edit.js

```diff
@@ -37,7 +37,7 @@ export default function SeparatorEdit( { attributes, setAttributes } ) {
 			'has-css-opacity': opacity === 'css',
 			'has-alpha-channel-opacity': opacity === 'alpha-channel',
 		},
-		colorProps.classname
+		colorProps.className
```

Comment:
@glendaviesnz am I onto something here, that this was only a small typo, or am I missing something intentional?

Reply:
Definitely a typo, thanks for sorting, tested well for me and this brings it back into line with the save.js method.